bagging.md

```diff
@@ -1,22 +1,32 @@
 # Baggers
 
 - Getting set up as a Bagger
-  - Talk to your DataRescue guide to make sure you can become a Bagger.
+  - Apply to become a Bagger 
```

Comment:
We have a google form, but I'm not thrilled about it. But one piece of feedback that keeps emerging  is the importance of chain of custody. So, we actually need to have names associated with the people who have control over the data after it's dowloaded. we want to leave harvesting as possible to be done by anyone, but that checking, bagging, and describing roles should be able to do their work only after they've filled out a google form to identify themselves. Does that make sense? But, basic answer is a google form.